<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrievePortalDataWidget</class>
 <widget class="QWidget" name="RetrievePortalDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="manifestGroupBox">
     <layout class="QFormLayout" name="manifestFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSearch">
        <property name="text">
         <string>Search file:  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDatasetID">
        <property name="text">
         <string>Dataset ID:  </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditDatasetID"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableViewSearchResult">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDownload">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExportVTK">
       <property name="text">
        <string>Export VTK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAnalyse">
       <item>
        <property name="text">
         <string>heart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lung</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>stomach</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kidney</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>colon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>skin</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnalyse">
       <property name="text">
        <string>Analyse</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeViewFileBrowser">

    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
         <item>
      <widget class="QPushButton" name="pushButtonDone">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
